### PR TITLE
Create /updates endpoint to list all the UpdateRequests

### PR DESCRIPTION
**Pull request by txus**
`/api/v1/updates` returns a list of all `UpdateRequest` objects.

### CODE REVIEW

File: core/src/main/scala/org/genivi/sota/core/data/UpdateRequest.scala [hunk 1]
```diff
@@ -10,6 +10,37 @@ import org.joda.time.{Interval, DateTime}
 
 case class UpdateRequest( id: UUID, packageId: PackageId, creationTime: DateTime, periodOfValidity: Interval, priority: Int )
 
+object UpdateRequest {
```
Comment (koshelev) [inline core/src/main/scala/org/genivi/sota/core/data/UpdateRequest.scala]: Spray json is deprecated. Please use circe instead

File: core/src/test/scala/org/genivi/sota/core/UpdateRequestSpec.scala [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    val resource = new UpdateRequestsResource(db, new UpdateService())
+  }
+
+  import UpdateRequest._
+  import spray.json.DefaultJsonProtocol._
+  import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+
+  property("Update requests can be listed")  {
+    new Service() {
+      forAll(Gen.listOf(updateRequestGen(PackageIdGen))) { (requests: Seq[UpdateRequest]) =>
```
Comment (koshelev) [inline core/src/test/scala/org/genivi/sota/core/UpdateRequestSpec.scala]: It should be something wrong with the test. It should not be possible to persist UpdateRequest if the package is not in db. Or am I missing smth.?

Comment (stevana) [inline core/src/test/scala/org/genivi/sota/core/UpdateRequestSpec.scala]: For the record this test, before this commit, often fails for me locally:

```
[info] UpdateServiceSpec:
[info] - decline if package not found (10 seconds, 39 milliseconds)
[info] - decline if some of dependencies not found (10 seconds, 156 milliseconds)
[info] - upload spec per vin *** FAILED *** (4 seconds, 27 milliseconds)
[info]   TestFailedException was thrown during property evaluation.
[info]     Message: The future returned an exception of type: java.sql.SQLIntegrityConstraintViolationException, with message: Duplicate entry '\xEF\x95\xD1\xDA\x84IM2\x8A\xC6 \x00[\xE0\x00M' for key 'PRIMARY'.
[info]     Location: (UpdateServiceSpec.scala:96)
[info]     Occurred when passed generated values (
[info]       arg0 = UpdateRequest(ef95d1da-8449-4d32-8ac6-20005be0004d,PackageId(Refined(yoshimi-dbg),Refined(1.2.4-1)),2015-09-15T11:06:35.161+02:00,2015-12-13T11:06:35.161+01:00/2016-02-10T11:06:35.161+01:00,10),
[info]       arg1 = Map() // 4 shrinks
[info]     )
```

**Comment by jerrytrieu** [comment]
Need rebase. Close.

**Pull request closed without merging**